app/utils.go

```diff
@@ -50,11 +54,15 @@ var DefaultConsensusParams = &abci.ConsensusParams{
 
 // Setup initializes a new EthermintApp. A Nop logger is set in EthermintApp.
 func Setup(isCheckTx bool, patchGenesis func(*EthermintApp, simapp.GenesisState) simapp.GenesisState) *EthermintApp {
-	db := dbm.NewMemDB()
+	return SetupWithDB(isCheckTx, patchGenesis, dbm.NewMemDB())
```

Comment:
I feel this improvement can be merged in another PR?

Reply:
extracted to https://github.com/evmos/ethermint/pull/1171